src/shared/ui/Input.tsx

```diff
@@ -38,6 +40,19 @@ export const InputFrame = styled(
           placeholderTextColor: '$lighterGrey',
           borderBottomColor: '#1e1d1d',
         },
+        survey: {
+          borderBottomWidth: 1,
+          borderBottomColor: '$mediumGrey',
+          width: '100%',
```

Comment:
Layout styles should go upper. See css properties ordering recomendations.

Reply:
Do you mean the following? https://tamagui.dev/docs/intro/props#order-is-important

Reply:
@anq83 Do you mean the following? https://tamagui.dev/docs/intro/props#order-is-important